src/helpers/getErc725yContract/getErc725yContract.ts

```diff
@@ -0,0 +1,45 @@
+import { BytesLike, Provider, Signer, Wallet, isAddress, isAddressable } from 'ethers';
+import { ERC725Y, ERC725Y__factory } from '../..';
+import { INTERFACE_IDS } from '@lukso/lsp-smart-contracts';
+
+export async function getErc725yContract(erc725y: ERC725Y): Promise<ERC725Y>;
+export async function getErc725yContract(
+    erc725y: ERC725Y,
+    signer: Signer | Wallet | Provider,
+): Promise<ERC725Y>;
+export async function getErc725yContract(
+    erc725y: BytesLike | string,
+    signer: Signer | Wallet | Provider,
+): Promise<ERC725Y>;
```

Comment:
I think these 2 can be merged into one

```suggestion
export async function getErc725yContract(
    erc725y: ERC725Y | BytesLike | string,
    signer: Signer | Wallet | Provider,
): Promise<ERC725Y>;
```